pkgs/development/libraries/quarto/default.nix

```
@@ -0,0 +1,63 @@
+{ stdenv
+, lib
+, pandoc
+, esbuild
+, deno
+, fetchurl
+, makeWrapper
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "quarto";
+  version = "1.0.38";
+
+  src = fetchurl {
+    url = "https://github.com/quarto-dev/quarto-cli/releases/download/v${finalAttrs.version}/quarto-${finalAttrs.version}-linux-amd64.tar.gz";
+    sha256 = "sha256-bqEGRX6ihyrgmIxqkV9jr/2QfWM5LWnsjoGpTqK9qOc=";
+  };
+
+  buildInputs = [
+      pandoc
+      deno
+      esbuild
+      makeWrapper
```

Please follow the formatting style in the contributing guide.

```suggestion
    pandoc
    deno
    esbuild
```
and at least makeWrapper goes to nativeBuildInputs. If pandoc is used to build the docs and/or esbuild to build js files, then that/those, too

---

Well, pandoc is a runtime dependency and the documentation does not clarify if they need esbuild only during build time or/and it is a runtime dependency.

---

Then we need to test this out